pyart/retrieve/comp_z.py

```diff
@@ -0,0 +1,109 @@
+"""
+Calculate the composite reflectivity 
+
+"""
+
+import copy
+import numpy as np
+from netCDF4 import num2date
+from pandas import to_datetime
+
+def composite_reflectivity(radar,rhv_filter=True,rhv_value=0.95,verbose=False):
```

Comment:
I would make this more generic to take in any gatefilter, and if there isn't one, you can create one with the parameters you mentioned (for rho_hv)